### PR TITLE
feat(tokens): Add colorNeutralBackgroundAlpha and colorNeutralStrokeAlpha tokens

### LINKED ISSUE #26335: [Feature]: Neutral Stroke Alpha + Neutral background alpha token
### Library

React Components / v9 (@fluentui/react-components)

### Describe the feature that you would like added

Requesting 3 new tokens
1. Neutral Stroke Alpha
image
https://www.figma.com/file/KB9oUjMKen2cKnyPG7RgdS/Design-tokens-superset?node-id=35689%3A78022&t=H2d1974RrD8ZZsfZ-4

2. Neutral background alpha
image
https://www.figma.com/file/KB9oUjMKen2cKnyPG7RgdS/Design-tokens-superset?node-id=35800%3A81943&t=WWBY5X9w9B5Upyu0-4

3. Neutral background alpha 2
image
https://www.figma.com/file/KB9oUjMKen2cKnyPG7RgdS/Design-tokens-superset?node-id=37910%3A79861&t=yOelZ78mc9V58Am0-4
### Have you discussed this feature with our team

Juraj Kapsiar

### Additional context

Neutral stroke alpha is a Subtle border treatment that designers can leverage to add slight definition to containers

image
Teams would like to use this in our left rail (Tree component), to bring our visuals closer to the Win 11 visual language

Neutral background alpha is for enabling native material (mica) to show through on desktop client for the left nav
Neutral background alpha 2 is for enabling native material (mica) to show through on desktop client for the canvas nav

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by miroslavstastny**
Adds `colorNeutralBackgroundAlpha`, `colorNeutralBackgroundAlpha2` and `colorNeutralStrokeAlpha` tokens

image

## Related Issue(s)

- Fixes #26335

### CODE REVIEW

File: packages/tokens/src/types.ts [hunk 1]
```diff
@@ -638,7 +640,6 @@ export type ShadowBrandTokens = {
 };
 
 export type Greys =
-  | 0
```
Comment (miroslavstastny) [inline packages/tokens/src/types.ts:641]: The two shades of grey have been removed by the token pipeline. Agreed on async, those are not exported (global colors) and never used (As Grey.0 is Black and Grey.100 is White).

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/Avatar-mount_base.html">1278</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/Avatar-mount.html">1277</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/Button-mount_base.html">934</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/Button-mount.html">926</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/Field-mount_base.html">1952</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/Field-mount.html">1898</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/FluentProvider-mount_base.html">1547</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/FluentProvider-mount.html">1556</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/FluentProviderWithTheme-mount_base.html">578</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/FluentProviderWithTheme-mount.html">575</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">539</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">545</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">573</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">577</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/InfoButton-mount_base.html">519</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/InfoButton-mount.html">522</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/MakeStyles-mount_base.html">1956</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/MakeStyles-mount.html">1970</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/Persona-mount_base.html">2828</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/Persona-mount.html">2803</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/SpinButton-mount_base.html">2280</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27034/perf-test-react-components/SpinButton-mount.html">2303</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                |     Baseline (minified/GZIP) |                           PR |                                                                                                                                                                                                  Change |
| :--------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: | ---------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> |  `64.09 kB`<br />`17.544 kB` | `64.148 kB`<br />`17.554 kB` |  `58 B` <br />`10 B`  |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>               | `35.049 kB`<br />`11.533 kB` | `35.107 kB`<br />`11.543 kB` |  `58 B` <br />`10 B`  |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                     |  `30.908 kB`<br />`6.714 kB` |  `31.122 kB`<br />`6.744 kB` | `214 B` <br />`30 B`  |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                   |  `17.776 kB`<br />`5.141 kB` |  `17.832 kB`<br />`5.151 kB` |  `56 B` <br />`10 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `85.876 kB`<br />`26.019 kB` |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |  `87.465 kB`<br />`21.569 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `51.192 kB`<br />`14.043 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |    `15.42 kB`<br />`6.167 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |  `67.414 kB`<br />`18.551 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `23.113 kB`<br />`7.069 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `24.002 kB`<br />`7.374 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |   `25.438 kB`<br />`7.254 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `36.888 kB`<br />`9.412 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |  `44.063 kB`<br />`10.885 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |  `41.568 kB`<br />`10.742 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |  `49.997 kB`<br />`12.321 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |  `55.252 kB`<br />`11.331 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `83.548 kB`<br />`23.894 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `78.317 kB`<br />`22.433 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |    `9.045 kB`<br />`3.806 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |   `10.984 kB`<br />`4.509 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `9.852 kB`<br />`4.159 kB` |
| <samp>react-checkbox</samp> <br /> <abbr title='bundle-size/Checkbox.fixture.js'>Checkbox</abbr>                                                                              |  `33.091 kB`<br />`10.319 kB` |
| <samp>react-checkbox</samp> <br /> <abbr title='bundle-size/CheckboxField.fixture.js'>CheckboxField</abbr>                                                                    |  `39.996 kB`<br />`12.206 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `85.509 kB`<br />`27.437 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/ComboboxField.fixture.js'>ComboboxField</abbr>                                                                    |  `81.926 kB`<br />`27.213 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |  `84.196 kB`<br />`27.191 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> |  `202.309 kB`<br />`56.37 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                                    |  `89.808 kB`<br />`26.701 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |     `17.11 kB`<br />`6.18 kB` |
| <samp>react-field</samp> <br /> <abbr title='bundle-size/Field.fixture.js'>Field</abbr>                                                                                       |   `17.673 kB`<br />`6.543 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |   `11.259 kB`<br />`4.473 kB` |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoButton.fixture.js'>InfoButton</abbr>                                                                        | `127.411 kB`<br />`38.824 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |    `22.94 kB`<br />`7.214 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/InputField.fixture.js'>InputField</abbr>                                                                             |  `32.819 kB`<br />`10.111 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |    `9.822 kB`<br />`4.066 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `12.306 kB`<br />`5.073 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          |  `126.86 kB`<br />`38.831 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `129.929 kB`<br />`39.345 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                                                               |   `11.214 kB`<br />`4.271 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                                 |  `58.207 kB`<br />`15.986 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 | `114.483 kB`<br />`35.249 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |   `11.377 kB`<br />`4.179 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                                                                    |    `23.791 kB`<br />`8.73 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressBar.fixture.js'>ProgressBar</abbr>                                                                        |     `13.59 kB`<br />`5.24 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressField.fixture.js'>ProgressField</abbr>                                                                    |   `23.885 kB`<br />`8.385 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `17.116 kB`<br />`6.332 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                                                                       |  `32.676 kB`<br />`10.616 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                                                             |   `14.885 kB`<br />`5.933 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroupField.fixture.js'>RadioGroupField</abbr>                                                                   |    `24.963 kB`<br />`9.07 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |   `24.145 kB`<br />`8.271 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/SelectField.fixture.js'>SelectField</abbr>                                                                          |  `33.384 kB`<br />`10.779 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |  `33.091 kB`<br />`10.555 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/SliderField.fixture.js'>SliderField</abbr>                                                                          |  `42.922 kB`<br />`13.465 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                                        |    `32.83 kB`<br />`9.864 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButtonField.fixture.js'>SpinButtonField</abbr>                                                              |  `41.766 kB`<br />`12.254 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |    `20.62 kB`<br />`6.682 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |   `28.225 kB`<br />`8.755 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/SwitchField.fixture.js'>SwitchField</abbr>                                                                          |   `35.01 kB`<br />`10.655 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                                 | `144.785 kB`<br />`39.447 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |  `43.776 kB`<br />`12.063 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 |   `134.4 kB`<br />`34.514 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                                         |  `82.316 kB`<br />`20.576 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                                   |  `81.637 kB`<br />`20.386 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `12.261 kB`<br />`4.819 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |    `15.571 kB`<br />`5.23 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                              |   `26.336 kB`<br />`8.585 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/TextareaField.fixture.js'>TextareaField</abbr>                                                                    |  `36.537 kB`<br />`11.504 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>                                                        |            `69 B`<br />`89 B` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `46.328 kB`<br />`16.212 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/5177d2a76ad0c003f03a883022ff5cee62c3e7e3'>5177d2a76ad0c003f03a883022ff5cee62c3e7e3</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: packages/react-components/react-theme/stories/Theme/colors/ThemeColors.stories.tsx [hunk 2]
```diff
@@ -98,7 +98,7 @@ const ColorButton: React.FunctionComponent<
 );
 
 const neutralTokens = (Object.keys(theme.webLight) as Array<keyof Theme>).filter(tokenName =>
-  tokenName.match(/^color(?!Palette).*/),
+  tokenName.match(/^colorNeutral(Background|Stroke)Alpha.*/),
```
Comment (khmakoto) [inline packages/react-components/react-theme/stories/Theme/colors/ThemeColors.stories.tsx:101]: If you want to be more specific but still include all colors that were previously included you should do:

```suggestion
  tokenName.match(/^color(Neutral|Brand|CompoundBrand|Subtle|Transparent)?(Background|Foreground|Scrollbar|Shadow|Stencil|Stroke)(Alpha|Overlay)?.*/),
```

You can check that this works [here](https://regex101.com/r/jNyPbX/1)

File: change/@fluentui-react-migration-v8-v9-daf3e7a9-933f-46b2-8b57-d229db417da9.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add colorNeutralBackgroundAlpha and colorNeutralStrokeAlpha tokens",
```
Comment (Hotell) [inline change/@fluentui-react-migration-v8-v9-daf3e7a9-933f-46b2-8b57-d229db417da9.json:3]: This is probably a feature addition no ?

```suggestion
  "comment": "feat: Add colorNeutralBackgroundAlpha and colorNeutralStrokeAlpha tokens",
```

File: change/@fluentui-react-migration-v8-v9-daf3e7a9-933f-46b2-8b57-d229db417da9.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (Hotell) [inline change/@fluentui-react-migration-v8-v9-daf3e7a9-933f-46b2-8b57-d229db417da9.json:2]:
```suggestion
  "type": "minor",
```

File: packages/react-components/react-theme/stories/Theme/colors/ThemeColors.stories.tsx (hunk 2 above)
Comment (miroslavstastny) [reply packages/react-components/react-theme/stories/Theme/colors/ThemeColors.stories.tsx:101]: Actually, this is just a debugging code which should have not been pushed.

Comment (miroslavstastny) [reply packages/react-components/react-theme/stories/Theme/colors/ThemeColors.stories.tsx]: removed

**Review by bsunderhus — APPROVED** [review]

**Review by layershifter — APPROVED** [review]

**Review by khmakoto — APPROVED** [review]

**Pull request merged**